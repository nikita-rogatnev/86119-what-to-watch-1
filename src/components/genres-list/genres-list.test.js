import React from 'react';
import renderer from 'react-test-renderer';

import GenresList from './genres-list';

it(`GenresList renders correctly`, () => {
  const tree = renderer.create(<GenresList/>).toJSON();
  expect(tree).toMatchSnapshot();
});
