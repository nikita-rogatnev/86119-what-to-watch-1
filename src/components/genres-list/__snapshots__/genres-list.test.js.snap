// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresList renders correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      onClick={[Function]}
    >
      All genres
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      onClick={[Function]}
    >
      comedies
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      onClick={[Function]}
    >
      documentaries
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      onClick={[Function]}
    >
      dramas
    </a>
  </li>
</ul>
`;
